src/config.cpp

```diff
@@ -147,6 +147,10 @@ struct Inputs_Element : JSON::Element {
       v_.cross_past_key_names = value;
     } else if (name == "cross_past_value_names") {
       v_.cross_past_value_names = value;
+    } else if (name == "curr_seq_length") {
```

Comment:
We should try to use the full-length names as the labels in the config. The abbreviated names can be the values of the labels.